src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -236,6 +236,31 @@
 
         {% if excluded_ajax_paths is defined %}
             if (window.XMLHttpRequest && XMLHttpRequest.prototype.addEventListener) {
+                if (window.fetch) {
+                    var oldFetch = window.fetch;
+                    window.fetch = function () {
+                        var promise = oldFetch.apply(null, arguments);
+                        var stackElement = {
+                            loading: true,
+                            error: false,
+                            url: arguments[0],
```

Comment:
you need to match the URL against the excluded ajax paths before adding it in the request stack